app/src/main/java/org/jellyfin/mobile/model/sql/entity/UserEntity.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.model.sql.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import org.jellyfin.mobile.model.sql.entity.UserEntity.Key.SERVER_ID
+import org.jellyfin.mobile.model.sql.entity.UserEntity.Key.TABLE_NAME
+import org.jellyfin.mobile.model.sql.entity.UserEntity.Key.USER_ID
+
+@Entity(tableName = TABLE_NAME, primaryKeys = [SERVER_ID, USER_ID])
+data class UserEntity(
+    @ColumnInfo(name = SERVER_ID) val serverId: Long,
```

Comment:
```suggestion
    @ColumnInfo(name = SERVER) val server: Long,
```

IMO if we refer to a table we can just use the name of the table since we'll always point to the primary key of the referred table.

Reply:
I disagree, since this isn't an instance of `ServerEntity`. It only makes sense in classes like `ServerUser`, where it's actually mapped to that type.